src/Templates/NewPowerShellScriptModule/test/Module.T.ps1

```diff
@@ -1,4 +1,5 @@
-$ModuleManifestName = '<%=$PLASTER_PARAM_ModuleName%>.psd1'
+$ModuleName = '<%=$PLASTER_PARAM_ModuleName%>'
+$ModuleManifestName = "$ModuleName.psd1"
```

Comment:
Can you change the value to `"${ModuleName}.psd1"?  It reads a bit better.  Thanks.

Reply:
I've been thinking about that too initially, but ScriptAnalyzer thinks that the ```$ModuleName```posh variable is not in use and tags it accordingly. So, should we still go with ```${ModuleName}```posh instead?

Reply:
@rkeithhill, I'm afraid that I'm going to need your answer before proceeding :)

Reply:
If my suggested change triggers a PSSA warning then no, don't change it.